### PR TITLE
Fan Page Part 3

### PR DESCRIPTION
**Pull request by kmbrandt94**

### CODE REVIEW

File: index-alt.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        <ol class="list">
+            <li value="1"><a href="http://www.elderscrolls.com/arena/">Arena</a></li>
+            <li value="2"><a href="http://www.elderscrolls.com/daggerfall/">Daggerfall</a></li>
+            <li value="3"><a href="http://www.elderscrolls.com/morrowind/">Morrowind</a></li>
+            <li value="4"><a href="http://www.elderscrolls.com/oblivion/">Oblivion</a></li>
+            <li value="5"><a href="http://www.elderscrolls.com/skyrim/">Skyrim</a></li>
+            <li class="feature">They also released <a href="http://www.elderscrollsonline.com/en-us/agegate">The Elder Scrolls Online</a> more recently.</li>
+        </ol>
+        <div>
+            <a href="tamriel.png"><img src="tamriel.png" height="200" width="350" alt="view a larger picture"></a>
```
Comment (mbMosman) [inline index-alt.html]: thumbnail link to larger picture :heavy_check_mark:

File: style-alt.css [hunk 2]
```diff
@@ -0,0 +1,47 @@
+body {
+    background-color: #978E43;
+    color: #350608;
+    text-align: center;
+    font-family: sans-serif;
+    background-image: url(texture.jpg);
```
Comment (mbMosman) [inline style-alt.css]: background image? :heavy_check_mark:

File: style-alt.css [hunk 3]
```diff
@@ -0,0 +1,47 @@
+body {
+    background-color: #978E43;
+    color: #350608;
+    text-align: center;
+    font-family: sans-serif;
```
Comment (mbMosman) [inline style-alt.css]: You should use more interesting fonts with fall backs. Check out CSS Font Stack for help on setting them up.

File: style-alt.css [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+.feature {
+    color: #C73B0B;
+    list-style-type: circle;
+}
+
+.list {
+    text-align: center;
+    list-style-position: inside;
+    list-style-type: upper-roman;
```
Comment (mbMosman) [inline style-alt.css]: different list markers? :heavy_check_mark:

File: style-alt.css [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+}
+
+#altpage {
+    font-weight: bold;
+    font-size: large;
+}
+
+#footer {
+    font-size: x-small;
```
Comment (mbMosman) [inline style-alt.css]: smaller footer font? :heavy_check_mark:

File: style-alt.css (hunk 4 above)
Comment (kmbrandt94) [inline style-alt.css]: they are, one is lower case and one is upper

File: style-alt.css (hunk 2 above)
Comment (kmbrandt94) [inline style-alt.css]: is the texture not good enough? should I use another image?

**Pull request closed without merging**